src/io.jl

```diff
@@ -328,6 +340,9 @@ Write out a julia `Array` as a PNG image.
 - `background`: optional background color to be stored in the `bKGD` chunk. Only meaningful for transparent images.
     Valid values are `nothing` for no background, `UInt8` as a palette index for palleted images,
     `Gray` for grayscale images and `RGB` for true color images.
+- `dpi`: stores the pixel density given in dots per inch into the `pHYs` chunk as pixels per meter.
+    The density is given as `dpi` for ease of use as pixels per meter is an uncommon format. If set to `nothing`,
+    no pixel density is written. If set to a 2-element tuple, a different density is written for x and y, respectively.
```

Comment:
Pixels per meter seems to be native to PNG (cf. https://www.w3.org/TR/PNG-Chunks.html), *and* it is using an SI unit, so even though DPI may be more common, why not at least support both a `pixels_per_meter` and a `dots_per_inch` keyword?

Reply:
would also be possible, I just don't think anybody will really use that :)